zstacklib/zstacklib/utils/lvm_range.py

```
@@ -0,0 +1,264 @@
+    for part in report.get("report", []):
+        rows.extend(part.get(section, []))
+    return rows
+
+
+def _number(value, field):
+    if value is None:
+        raise ValueError("missing numeric field[%s]" % field)
+    try:
+        return _long(str(value).strip())
+    except (TypeError, ValueError):
+        raise ValueError("invalid numeric field[%s:%s]" % (field, value))
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)